ui/packages/consul-ui/docs/hds.mdx

```diff
@@ -4,50 +4,50 @@ title: Hashicorp Design System
 
 # Hashicorp Design System
 
-The application integrates setup that make it possible to use the [Hashicorp Design System (HDS)](https://github.com/hashicorp/design-system) in the application.
+The application integrates setup that make it possible to use the [Hashicorp Design System (HDS)](https://github.com/hashicorp/design-system) in the application. The application also
+bundles [TailwindCSS](https://tailwindcss.com/) to follow a utility-based CSS approach that mixes well with HDS utility classes. Tailwind is expected to be used
+as a supplement to HDS's utility classes, and should mainly be used for layouting.
 
 ## Design Tokens
 
-HDS ships a set of [design tokens](https://design-system-components-hashicorp.vercel.app/foundations/tokens), which are implemented as CSS custom properties.
-To make it easy to work with these design tokens without having to work with verbose CSS properties all over the place, we have setup a [TailwindCSS](https://tailwindcss.com/)-configuration that makes the color tokens
-available via functional [utility classes](https://tailwindcss.com/docs/customizing-colors).
-
 ### Colors
 
-You can work with any color from the HDS by prefixing colors with the `hds-`-prefix. The `hds-`-prefix makes it easy to see what classes are auto-generated from HDS. When we wanted to color a header tag with the `consul-brand`-color we could do it like this for example:
+To make sure we follow HDS-guidelines, we disabled Tailwind's color utilities completely. Instead of relying on Tailwind for colors, you should be working with the colors from HashiCorp-Design-System and follow the [guidance in the docs](https://design-system-components-hashicorp.vercel.app/foundations/colors) on how to use them:
 
 ```hbs preview-template
-<h2 class="text-hds-consul-brand">
+<h2 class='hds-foreground-strong'>
   HDS is awesome
 </h2>
 ```
 
-Regular tailwindCss colors aren't available. You can only use colors defined in HDS with this pattern.
+Regular TailwindCSS colors aren't available.
 
 ```hbs preview-template
-<h2 class="text-red-400">
+<h2 class='text-red-400'>
   TailwindCSS colors won't work
 </h2>
 ```
 
-### Other tokens
+### HDS tokens
```

Comment:
I am worried this may lead to confusion. [In HDS we call `tokens` the key/value pairs in code](https://design-system-components-hashicorp.vercel.app/foundations/tokens) (generally, CSS custom props). What you're describing here are probably more similar to the the "foundation" (or "core") styles. They include colors, typography, elevation, and focus ring, and they all can be expressed via CSS helper/utility classes.